docs/profiles/0.5-DRAFT/process_run_crate/index.md

```diff
@@ -13,19 +13,19 @@ title: Process Run Crate
 * Profile Crate: [ro-crate-metadata.json](ro-crate-metadata.json) [ro-crate-preview.html](ro-crate-preview.html)
 * Extends:
   - [RO-Crate 1.1 specification](https://w3id.org/ro/crate/1.1)
-* JSON-LD context: <https://w3id.org/ro/terms/workflow-run>
+* JSON-LD context: <https://w3id.org/ro/terms/workflow-run/context.json>
```

Comment:
Should we better use just `/context` in the PID and `context.jsonld` in the repository so its content type will be correct?

Reply:
I think these had misleading file extensions in roterms repository..

Reply:
Fixed in https://github.com/ResearchObject/ro-terms/commit/ae5bfb8eb7e07a8ae35b0441dfc739479e445da2
and https://github.com/perma-id/w3id.org/pull/4168